### PR TITLE
Fix issue 25690 update Getting started-logging in page with missing screenshot

### PR DESCRIPTION
**Pull request by rghara**
## Hey, I just made a Pull Request!

In https://backstage.io/docs/getting-started/logging-in there is a reference to an image of the login screen but the image was missing.
1- Added an image of the login screen to docs/assets/getting-started/login-screen.png
2- Added reference to image to docs/getting-started/logging-in.md

expected

#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [X] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [X] Screenshots attached (for UI changes)
- [X] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: docs/assets/getting-started/login-screen.png
Comment (awanlin) [inline docs/assets/getting-started/login-screen.png:1]: We should use a screen shot from what you see after using `npx @backstage/create-app@latest`, this isn't what most people will see at that stage.

**Review by awanlin — CHANGES_REQUESTED** [review]
Good catch @rghara, thanks for helping get this fixed. Just left a comment about the image itself. 👍

File: docs/assets/getting-started/login-screen.png
Comment (rghara) [reply docs/assets/getting-started/login-screen.png:1]: Sounds good. I just have the getting started version. I haven't deployed a standalone backstage yet. I will be working on that and will update.

Sorry, my very first contribution :)

Comment (awanlin) [reply docs/assets/getting-started/login-screen.png:1]: No worries, happy to help you along 👍  

The getting started version should be from using `npx @backstage/create-app@latest`. You should only see the Guest option at this point.

Comment (awanlin) [reply docs/assets/getting-started/login-screen.png:1]: Sorry, you should have GitHub and Guest, now I feel like I need to double check this myself 🤔

Comment (rghara) [reply docs/assets/getting-started/login-screen.png:1]: Working on it right now using "npx @backstage/create-app@latest". Yes, you are right. We should see the Github option.

Comment (rghara) [reply docs/assets/getting-started/login-screen.png:1]: @awanlin For some reason if I follow the instructions and add the Github option in packages/app/src/App.tsx it doesn't show Guest anymore. Below is what I ended up with. Does this look more like it?

image

Comment (awanlin) [reply docs/assets/getting-started/login-screen.png:1]: Perfect, thanks!

**Review by awanlin — APPROVED** [review]
Awesome work, again thanks for catching and fixing this @rghara 🚀

**Comment by awanlin** [comment]
@rghara we just need a review from one of the maintainers now, they will also merge it 👍

**Comment by rghara** [comment]
@awanlin I appreciate your support!

**Review by benjdlambert — APPROVED** [review]
Thanks!

**Pull request merged**
